nix/grafana.nix

```diff
@@ -75,6 +75,24 @@ in
       '';
     };
 
+    providers = lib.mkOption {
+      type = types.listOf yamlFormat.type;
+      description = "List of dashboard providers to configure.";
```

Comment:
how about linking <https://grafana.com/docs/grafana/latest/administration/provisioning/#dashboards> in the description? The user will have idea of how the schema looks like

Reply:
Good idea. I'll add a similar link for the datasources schema.